### PR TITLE
rename git sha related

### PR DESCRIPTION
**Pull request by 9bany**
redisGitSHA1 -> serverGitSHA1  
redisGitDirty -> serverGitDirty 
redisBuildId -> serverBuildId 
redisBuildIdRaw -> serverBuildIdRaw 
redisBuildIdString -> serverBuildIdString

https://github.com/valkey-io/valkey/issues/144 https://github.com/valkey-io/valkey/issues/170

### CODE REVIEW

File: src/server.c [hunk 1]
```diff
@@ -5585,9 +5585,9 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
 
         info = sdscatfmt(info, "# Server\r\n" FMTARGS(
             "redis_version:%s\r\n", REDIS_VERSION,
-            "redis_git_sha1:%s\r\n", redisGitSHA1(),
-            "redis_git_dirty:%i\r\n", strtol(redisGitDirty(),NULL,10) > 0,
-            "redis_build_id:%s\r\n", redisBuildIdString(),
+            "redis_git_sha1:%s\r\n", serverGitSHA1(),
+            "redis_git_dirty:%i\r\n", strtol(serverGitDirty(),NULL,10) > 0,
+            "redis_build_id:%s\r\n", serverBuildIdString(),
```
Comment (zuiderkwast) [inline src/server.c:5590]: We might cause a merge conflict on these lines for #170. I can't see a PR for that yet so I'm assuming it's not started yet.

**Review by zuiderkwast — APPROVED** [review]

File: src/server.c (hunk 1 above)
Comment (daniel-house) [reply src/server.c:5590]: I just finished the code and am waiting for the test to complete.

Comment (daniel-house) [reply src/server.c:5590]: all tests passed - by the way, I also changed the contents of the strings, so, for example,
~~~
"redis_git_sha1:%s\r\n", serverGitSHA1(),
~~~
became
~~~
"valkey_git_sha1:%s\r\n", valkeyGitSHA1(),
~~~
and I updated the printver.tcl

What now?

Comment (zuiderkwast) [reply src/server.c:5590]: @daniel-house Where is your PR?

Comment (zuiderkwast) [reply src/server.c:5590]: I'm sorry I caused you merge conflicts. I hope it will be easy to solve though. it's just a few lines.

**Pull request merged**